Modal.html

```diff
@@ -0,0 +1,72 @@
+            }
+            }
+        }
+        }
+    </script>
+    <title>Modal</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+<div class="relative z-10" aria-labelledby="modal-title" role="dialog" aria-modal="true">
+    <!-- Background backdrop, show/hide based on modal state. -->
+    <!-- Entering: "ease-out duration-300" -->
+    <!-- From: "opacity-0" -->
+    <!-- To: "opacity-100" -->
+    <!-- Leaving: "ease-in duration-200" -->
+    <!-- From: "opacity-100" -->
+    <!-- To: "opacity-0" -->
+    <div class="fixed inset-0 bg-gray-500 bg-opacity-75 transition-opacity" aria-hidden="true"></div>
+
+    <div class="fixed inset-0 z-10 w-screen overflow-y-auto">
+        <div class="flex min-h-full items-end justify-center p-4 text-center sm:items-center sm:p-0">
+            <!-- Modal panel, show/hide based on modal state. -->
+            <!-- Entering: "ease-out duration-300" -->
+            <!-- From: "opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95" -->
+            <!-- To: "opacity-100 translate-y-0 sm:scale-100" -->
+            <!-- Leaving: "ease-in duration-200" -->
+            <!-- From: "opacity-100 translate-y-0 sm:scale-100" -->
+            <!-- To: "opacity-0 translate-y-4 sm:translate-y-0 sm:scale-95" -->
+            <div class="relative transform overflow-hidden rounded-lg bg-white text-left shadow-xl transition-all sm:my-8 sm:w-full sm:max-w-lg">
+                <div class="bg-white px-4 pb-4 pt-5 sm:p-6 sm:pb-4">
+                    <div class="sm:flex sm:items-start">
+                        <div class="mx-auto flex h-12 w-12 flex-shr class=" solid-green-100"ink-0 items-center justify-center rounded-full bg-green-100 sm:mx-0 sm:h-10 sm:w-10">
+                            <i class="fa fa-music" aria-hidden="true"></i>
+                        </div>
+                        <div class="mt-3 text-center sm:ml-4 sm:mt-0 sm:text-left">
+                            <h3 class="text-base font-semibold leading-6 text-gray-900" id="modal-title">Song Title</h3>
+                            <div class="mt-2">
+                                <p class="text-sm text-gray-500">Artist name</p>
+                                <hr>
+                                <p class="text-sm text-gray-500">Albumn name</p>
+                                <hr>
+                                <p class="text-sm text-gray-500">Wikipedia details-Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nemo perspiciatis repudiandae maxime? Nihil voluptates voluptatum quisquam nobis tenetur magnam sed, nesciunt cum omnis doloremque laborum et, excepturi inventore perferendis necessitatibus!</p>
+                                <hr>
+                                <p class="text-sm text-gray-500">Discography</p>
+                                <hr>
+                                <p class="text-sm text-gray-500">Similar Artists</p>
```

Comment:
Looks good, could I get you to update these `<p>` elements to each have `id`s assigned to them? This will make it possible for us to plug the values into here once we get the data from the API.

Reply:
No problem, will update